zeebe/broker/src/main/java/io/camunda/zeebe/broker/exporter/stream/ExporterDirector.java

```diff
@@ -82,8 +83,12 @@ public final class ExporterDirector extends Actor implements HealthMonitorable,
   private final ExporterMode exporterMode;
   private final Duration distributionInterval;
   private ExporterStateDistributionService exporterDistributionService;
+  private ScheduledTimer exporterDistributionTimer;
   private final int partitionId;
   private final EventFilter positionsToSkipFilter;
+  // When idle, exporter director is not exporting any records because no exporters are configured.
+  // The actor is still running, but it is not actively doing any work.
+  private boolean idle;
```

Comment:
It can be ACTIVE and idle meaning it is currently the leader, but idle. Or it can be PASSIVE and idle meaning it is the follower and idle. We have to remember whether it was leader or follower when switch back from idle. 
It is for the same reason, we cannot put in the ExporterPhase. We have to remember whether it was paused or exporting when switching back from idle.